index.html

```diff
@@ -10,229 +10,233 @@
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                            <li>Ruby on rails</li>
+                        </ul>
+                        <a class="button" href="">See project</a>
+                    </div>
+                </div>
+    
+                <div class="card">
+                    <div class="screenshot">
+                        <img src="/Images/placeholder.png" alt="screenshot">
+                    </div>
+                    
+                    <div class="info">
+                        <h3 class="text">
+                            Multi-Post Stories Gain+Glory
+                        </h3>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Ruby on rails</li>
+                        </ul>
+                        <a class="button" href="">See project</a>
+                    </div>
+                </div>
+            </div>
 
-        <section class="card">
-            <section class="screenshot">
-                <img src="/Images/placeholder.png" alt="screenshot">
-            </section>
-            
-            <section class="info">
-                <h3 class="text">
-                    Multi-Post Stories Gain+Glory
-                </h3>
-                <ul class="languages">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>Javascript</li>
-                </ul>
-                <a class="button" href="">See project</a>
-            </section>
         </section>
 
-        <section class="card">
-            <section class="screenshot">
-                <img src="/Images/placeholder.png" alt="screenshot">
-            </section>
-            
-            <section class="info">
-                <h3 class="text">
-                    Multi-Post Stories Gain+Glory
-                </h3>
-                <ul class="languages">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>Javascript</li>
-                    <li>Ruby on rails</li>
-                </ul>
-                <a class="button" href="">See project</a>
-            </section>
-        </section>
 
-        <section class="card">
-            <section class="screenshot">
-                <img src="/Images/placeholder.png" alt="screenshot">
-            </section>
-            
-            <section class="info">
-                <h3 class="text">
-                    Multi-Post Stories Gain+Glory
-                </h3>
-                <ul class="languages">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>Ruby on rails</li>
-                </ul>
-                <a class="button" href="">See project</a>
-            </section>
+        <section id="about">
+            <div class="aboutme">
```

Comment:
- [ ] As required by the previous reviewer, let's kindly please correct this gap:

![image](https://user-images.githubusercontent.com/57421823/149532559-b453b657-4a0a-4808-8af2-017efd65c2eb.png)

That way, your project will look a lot more like the given Figma design, as stated in the requirements 💯